packages/loot-core/src/platform/client/fetch/index.web.ts

```diff
@@ -78,11 +78,7 @@ export const init: T.Init = async function () {
   return new Promise(connectSocket);
 };
 
-export const send: T.Send = function (
-  name,
-  args,
-  { catchErrors = false } = {},
-) {
+export const send = function (name, args, { catchErrors = false } = {}) {
```

Comment:
Removed the `T.Send` because it was erroring on the return value (currently `Promise<unknown>` which doesn't match). To fix, we'd have to duplicate the types into this file. Happy to do so if the reviewer thinks it's worth it!